{"timestamp":"2025-08-05T00:45:28.152246","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-08-05T00:45:28.155748","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/etl_dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-08-05T00:45:33.356909Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:33.359697Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:33.361340Z","level":"info","event":"Current task name:run_python_etl","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:33.363126Z","level":"info","event":"Dag name:s3_to_snowflake_etl","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:33.365088Z","level":"info","event":"Starting CSV processing...","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:36.215630Z","level":"info","event":"Successfully read CSV from S3","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:36.397325Z","level":"info","event":"Processed 146 records","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:36.467358Z","level":"info","event":"First 5 rows:    expense_id date_of_expense   cost  category_id  vendor_id","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:36.469420Z","level":"info","event":"0           1      2025-05-30   7.29            1          1","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:36.470997Z","level":"info","event":"1           2      2025-05-30  50.00            2          2","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:36.472486Z","level":"info","event":"2           3      2026-05-13  20.00            3          3","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:36.474203Z","level":"info","event":"3           4      2025-07-23  22.13            4          4","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:36.475793Z","level":"info","event":"4           5      2025-06-15  45.67            5          5","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:36.468121","level":"info","event":"Snowflake Connector for Python Version: 3.16.0, Python Version: 3.12.11, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36","logger":"snowflake.connector.connection"}
{"timestamp":"2025-08-05T00:45:36.470330","level":"info","event":"Connecting to GLOBAL Snowflake domain","logger":"snowflake.connector.connection"}
{"timestamp":"2025-08-05T00:45:42.030029","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"ProgrammingError","exc_value":"000904 (42000): 01be27cd-0002-2044-0002-900600016066: SQL compilation error: error line 1 at position 86\ninvalid identifier '\"expense_id\"'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":877,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1164,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":217,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":240,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/etl_dag.py","lineno":111,"name":"process_csv_data"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/pandas_tools.py","lineno":600,"name":"write_pandas"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/cursor.py","lineno":1137,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/errors.py","lineno":279,"name":"errorhandler_wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/errors.py","lineno":334,"name":"hand_to_other_handler"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/errors.py","lineno":210,"name":"default_errorhandler"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-08-05T00:45:42.066557Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:42.067201Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:42.067748Z","level":"info","event":"Task:<Task(PythonOperator): run_python_etl>","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:42.068083Z","level":"info","event":"Failure caused by 000904 (42000): 01be27cd-0002-2044-0002-900600016066: SQL compilation error: error line 1 at position 86","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-05T00:45:42.068655Z","level":"info","event":"invalid identifier '\"expense_id\"'","chan":"stdout","logger":"task"}
